site/en/tutorials/text/text_generation.ipynb

```diff
@@ -622,7 +638,7 @@
       "source": [
         "For each character the model looks up the embedding, runs the GRU one timestep with the embedding as input, and applies the dense layer to generate logits predicting the log-likelihood of the next character:\n",
         "\n",
-        "![A drawing of the data passing through the model](images/text_generation_training.png)"
+        "![A drawing of the data passing through the model](https://github.com/masa-ita/tf-docs/blob/fix_word_usage_in_site_en_tutorials_text_text_generation/site/en/tutorials/text/images/text_generation_training.png?raw=1)"
```

Comment:
revert this URL change

Reply:
Thank you for your advice.  The URL seems to be changed accidentally when I save the file from Colab.  Fixed.